src/utils/searchForm.util.js

```diff
@@ -135,13 +151,87 @@ const getLastUpdate = function (req, modifier) {
     return dateString;
 };
 
+const formatDate = (res) => {
+    const dateString = res.meta && res.meta.lastUpdated ? res.meta.lastUpdated : '';
+    if (dateString === '') {
```

Comment:
Do you want to check if 'dateString' contains 'T' here too? If there isn't one then the code might error downstream

Reply:
Reworked the function completely

Reply:
Reworked the formatDate function